tasks/broad/ImputationTasks.wdl

```diff
@@ -4,18 +4,19 @@ task CalculateChromosomeLength {
   input {
     File ref_dict
     Int chrom
-    String ubuntu_docker
+    String ubuntu_docker = "ubuntu:20.04"
+    Int memory_mb = 2000
+    Int cpu = 1
+    Int disk_size_gb = ceil(2*size(ref_dict, "GiB")) + 5
   }
-
-  Int disk_size = ceil(2*size(ref_dict, "GiB")) + 5
-
   command {
     grep -P "SN:~{chrom}\t" ~{ref_dict} | sed 's/.*LN://' | sed 's/\t.*//'
   }
   runtime {
     docker: ubuntu_docker
-    disks: "local-disk " + disk_size + " HDD"
-    memory: "2 GiB"
+    disks: "local-disk " + disk_size_gb + " HDD"
```

Comment:
sometimes disk is being set (in other pipelines) using `disks: "local-disk ${disk_size_gb} HDD"` syntax. is one better than the other? do we care to make it all consistent? just curious!